src/pudl/transform/eia861.py

```diff
@@ -1241,7 +1248,7 @@ def demand_response(tfr_dfs):
     # these values have Nan BA values and should be deleted earlier.
     # thinking this might have to do with DR table weirdness between 2012 and 2013
     # will come back to this after working on the DSM table. Dropping dupes for now.
-    deduped_dr = _drop_dupes(tidy_dr, idx_cols)
+    deduped_dr = _drop_dupes(tidy_dr, 'Demand Response', idx_cols)
```

Comment:
this is just a nit but it's nice to have the names of the args when there are more than one arg (there are a few more instances of this throughout). It helps w/ clarity

```suggestion
    deduped_dr = _drop_dupes(df=tidy_dr, df_name='Demand Response', subsest=idx_cols)
```

Reply:
Good idea!